lib/honeydew/job_runner.ex

```diff
@@ -0,0 +1,87 @@
+defmodule Honeydew.JobRunner do
+  use GenServer
```

Comment:
What would you think about adding `@moduledoc false`, since this isn't part of Honeydew's public API?

Reply:
i always forget that...